src/__tests__/bulkUploads.int.test.ts

```diff
@@ -94,6 +95,95 @@ describe('/bulkUploads', () => {
 				);
 		});
 
+		it('returns all bulk uploads for administrative users', async () => {
+			const testUser = await loadTestUser();
+			const anotherUser = await createUser({
+				authenticationId: 'totallyDifferentUser@example.com',
+			});
+			await createBulkUpload({
+				fileName: 'foo.csv',
+				sourceKey: '96ddab90-1931-478d-8c02-a1dc80ae01e5-foo',
+				status: BulkUploadStatus.PENDING,
```

Comment:
Just out of curiosity, why make this a pending upload?

Reply:
Funsies (pending is the starting state of a bulk upload, so it's just as good a state as any really)